docs/untyped-blame.md

```diff
@@ -0,0 +1,219 @@
+    ```
+
+1.  Test that the new build works:
+
+    ```
+    ❯ bazel-bin/main/sorbet --track-untyped --print=untyped-blame -e 'T.unsafe(nil).foo'
+    No errors! Great job.
+    [{"path":"https://github.com/sorbet/sorbet/tree/master/rbi/sorbet/t.rbi","package":"<none>","owner":"T.class_of(T)","name":"unsafe","count":1}]
+    ```
+
+[Quickstart]: https://github.com/sorbet/sorbet/#quickstart
+
+## Collecting untyped blame information
+
+Sorbet collects untyped blame information by type checking a project, and
+dumping JSON information about untyped usages at the end. To use this new build
+of Sorbet to type check a project, change into your project's directory and run
+this command:
+
+```bash
+SRB_SORBET_EXE=path/to/bazel-bin/main/sorbet bundle exec srb tc \
+  --track-untyped --print=untyped-blame
+```
+
+**Note**: Replace `path/to/...` with the real path to the newly-built sorbet
+binary.
+
+The untyped blame information will output to stdout. To redirect it to a file,
+either use a Unix pipe, or use this form of the `--print` option, which will put
+the print output directly into a file (in this case, it will create the
+`/tmp/untyped-blame.json` file):
+
+```bash
+SRB_SORBET_EXE=path/to/bazel-bin/main/sorbet bundle exec srb tc \
+  --track-untyped --print=untyped-blame:/tmp/untyped-blame.json
+```
+
+## Interpreting the output
+
+The output will be a JSON array of JSON objects, where each object represents a
+definition, alongside how many usages of untyped blame to that definition.
+
+Each entry has these keys:
+
+- `owner`
+
+  The fully qualified name of the owner of the definition. For a method like
+  `A::B#foo`, the owner will be `A::B`. For a method like `A::B.foo`, the owner
+  will be `T.class_of(A::B)`.
+
+- `name`
+
+  The name of the definition. For a method like `A::B#foo`, the name will be
+  `foo`. The name will also be `foo` for a method like `A::B.foo` (though the
+  `owner` will be different, see above).
+
+- `path`
+
+  The path to the primary location of the definition. The primary location of
+  the definition is a Sorbet-internal heuristic that takes into account whether
+  the location is in an RBI file or not as well as the strictness level of that
+  file.
+
+- `count`
+
+  The number of usages of untyped that blame to this definition.
+
+- `package`
+
+  Will always be `<none>`, unless using Stripe's internal Ruby packaging
+  system. In that case, `package` will be the name of the package that owns this
+  definition.
+
+
+### Some special definitions
+
+There are some special definitions in the output that don't map to definitions
+defined anywhere in code.
+
+- `owner = <Magic>::<UntypedSource>`
+
+  Some usages of untyped arise because of `T.untyped` that arises internal to
+  Sorbet. For example, most existing limitation with [shape
+  types](https://sorbet.org/docs/shapes) arise because Sorbet will silently
+  introduce a usage of untyped.
```

Comment:
Not 100% sure about my suggestion, but there are too many `arise` in here :P

```suggestion
  Some usages of untyped arise because of `T.untyped`s that come from
  Sorbet internals. For example, most existing limitations with [shape
  types](https://sorbet.org/docs/shapes) exist because Sorbet silently
  introduces a usage of untyped.
```